content/plugins/aymanalhattami-slim-scrollbar.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Slim Scrollbar
+slug: aymanalhattami-slim-scrollbar
+author_slug: aymanalhattami
+categories: [panel-builder]
+description: A simple package to make the application's scrollbars slim.
+discord_url: 
+docs_url: https://github.com/aymanalhattami/filament-slim-scrollbar/blob/main/README.md
```

Comment:
Please fix this so that it points to a raw markdown file URL, like `https://raw.githubusercontent.com/filamentphp/spatie-laravel-media-library-plugin/3.x/README.md` in our example.

Reply:
README.md has been updated to use raw markdown file URL